NEWS.md

```diff
@@ -7,8 +7,9 @@ Changes in version 0.14
 
 **NOT RELEASED YET; STILL UNDER DEVELOPMENT.**
 
-* No changes recorded.
-
+* Explicitly require C++-11 language features when compiling Kyua to avoid
+  having to rely on Boost for features made available in C++-11, e.g.,
```

Comment:
The Boost comment here feels spurious. Kyua doesn't use Boost anywhere, so this mention comes out of the blue.